README.md

```diff
@@ -42,47 +42,51 @@ Containers-Toolkit is a Windows PowerShell module for downloading, installing, a
 
 **Option 1:**  Clone containers-toolkit into one of the folder locations in the `$env:PSModulePath` environment variable.
 
-1. To get a possible module path:
+1. **To get a possible module path:**
 
     ```PowerShell
     $env:PSModulePath -split ";"
     ```
 
-2. Clone the repo
+1. **Clone the repo**
 
     ```PowerShell
     cd <selected-module-path>
     git clone https://github.com/microsoft/containers-toolkit.git
     ```
 
-3. Import the module
+1. **Import the module**
 
     ```PowerShell
     Import-Module -Name containers-toolkit -Force
     ```
 
 **Option 2:** Clone containers-toolkit to a folder location of choice and add the new module location to the Windows PowerShell module path
 
-1. Clone the repo
+1. **Clone the repo**
 
     ```PowerShell
     git clone https://github.com/microsoft/containers-toolkit.git
     ```
 
-1. Add the directory to Windows PowerShell module path
+1. **Add the directory to Windows PowerShell module path**
 
     ```PowerShell
     $env:PSModulePath += ";<path-to-module-directory>"
     ```
 
-1. Install module dependencies
+1. **Install module dependencies**
+   - Install `ThreadJob` module
 
     ```powershell
     Install-Module -Name ThreadJob -Force
-    Install-Module -Name HNS -AllowClobber -Force
     ```
 
-1. Import the module
+    - Install `HNS` module
+
+        To install the HNS module. follow the instructions here [instructions](./docs/FAQs.md#2-new-hnsnetwork-command-does-not-exist)
```

Comment:
Can this be part of the script so that we shorted the number of steps it takes to setup?